upup/models/cloudup/resources/addons/authentication.aws/k8s-1.12.yaml.template

```
@@ -131,8 +131,8 @@ spec:
         {{- if or (not .Authentication.Aws.BackendMode) (contains "MountedFile" .Authentication.Aws.BackendMode) }}
         - --config=/etc/aws-iam-authenticator/config.yaml
         {{- end }}
-        {{- if or .Authentication.Aws.ClusterID (not (contains "MountedFile" .Authentication.Aws.BackendMode))}}
-        - --cluster-id={{ or .Authentication.Aws.ClusterID .MasterPublicName }}
+        {{- if or .Authentication.Aws.ClusterID (or (not .Authentication.Aws.BackendMode) (not (contains "MountedFile" .Authentication.Aws.BackendMode)))}}
```

Can this be simplified?

Can this be simplified?

```suggestion
        {{- if or .Authentication.Aws.ClusterID (not .Authentication.Aws.BackendMode) (not (contains "MountedFile" .Authentication.Aws.BackendMode))}}
```

---

Much clearer. Thanks!